src/NuGet.Clients/NuGetConsole.Host.PowerShell/PowerShellHost.cs

```diff
@@ -312,6 +313,20 @@ public void Initialize(IConsole console)
                         }
 
                         UpdateWorkingDirectory();
+
+                        if (!PowerShellLoaded)
+                        {
+                            var telemetryEvent = new TelemetryEvent("PowerShellLoaded", new Dictionary<string, object>
```

Comment:
Use a string constant or a readonly string:

```suggestion
                            var telemetryEvent = new TelemetryEvent(NuGetPowerShellUsageCollector.PowerShellLoaded, new Dictionary<string, object>
```

Reply:
Addressed.